### PR TITLE
vulnerabilities CVE report draft

### PR DESCRIPTION
**Pull request by radekkaluzik**
https://issues.redhat.com/browse/RHCLOUD-18160

### CODE REVIEW

File: browser/index.ts [hunk 1]
```diff
@@ -108,7 +108,7 @@ const generatePdf = async (
     path: pdfPath,
     format: 'a4',
     printBackground: true,
-    landscape: true,
+    // landscape: false,
```
[the remark this answers is not in the document]
Comment (radekkaluzik) [reply browser/index.ts:111]: Line removed. `landscape: false` doesn't work

File: templates/common/page.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export interface PageProps extends React.HTMLProps<HTMLDivElement> {
+  children?: React.ReactNode;
+}
+
+const Page: React.FunctionComponent<PageProps> = ({ children }: PageProps) => (
+  <div
+    style={{
```
[the remark this answers is not in the document]
Comment (radekkaluzik) [reply templates/common/page.tsx:9]: `Page.tsx` was renamed to `page.tsx`. I can see only `page.tsx` in the file structure.

[the remark this answers is not in the document]
Comment (Hyperkid123) [reply templates/common/page.tsx:9]: Well GH says that [templates/common/Page.tsx](https://github.com/RedHatInsights/pdf-generator/pull/6/files#diff-9a4a418af750ef16840ac846e9e7a590d06d13d0490896c57c0a6e784f835e7a) still exists :shallow_pan_of_food: . You sure you pushed all commits?

File: templates/common/Page.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export interface PageProps extends React.HTMLProps<HTMLDivElement> {
+  children?: React.ReactNode;
+}
+
+const Page: React.FunctionComponent<PageProps> = ({ children }: PageProps) => (
+  <div
+    style={{
```
[the remark this answers is not in the document]
Comment (radekkaluzik) [reply templates/common/Page.tsx:9]: Styling moved to `styles/styles.scss`

File: templates/common/page.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (radekkaluzik) [reply templates/common/page.tsx]: It should be removed by now. I pushed my changes minute ago.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**